src/storage/object.ts

```diff
@@ -465,4 +470,42 @@ export class ObjectStorage {
       })
     )
   }
+
+  /**
+   * Generates a signed url for uploading an object
+   * @param objectName
+   * @param url
+   * @param expiresIn seconds
+   * @param owner
+   */
+  async signUploadObjectUrl(objectName: string, url: string, expiresIn: number, owner?: string) {
+    // check as super user if the object already exists
+    let found
+    try {
+      found = await this.findObject(objectName, 'id', true)
```

Comment:
it would be better to do:

```
await this.asSuperUser().findObject(objectName, 'id')
```

instead of passing the the param in the function

Reply:
Do you mean that I have to introduce a new function `asSuperUser()` for `ObjectStorage` or should I simply use `this.db.asSuperUser()`?